templates/default/kserve-serving-runtimes.yml.erb

```diff
@@ -0,0 +1,313 @@
+    - autoSelect: true
+      name: sklearn
+      version: "0"
+    - autoSelect: true
+      name: xgboost
+      version: "1"
+    - autoSelect: true
+      name: lightgbm
+      version: "3"
+    - autoSelect: true
+      name: mlflow
+      version: "1"
+---
+apiVersion: serving.kserve.io/v1alpha1
+kind: ClusterServingRuntime
+metadata:
+  name: kserve-paddleserver
+spec:
+  annotations:
+    prometheus.kserve.io/path: /metrics
+    prometheus.kserve.io/port: "8080"
+  containers:
+    - args:
+        - --model_name={{.Name}}
+        - --model_dir=/mnt/models
+        - --http_port=8080
+      image: kserve/paddleserver:v0.10.0
+      name: kserve-container
+      resources:
+        limits:
+          cpu: "1"
+          memory: 2Gi
+        requests:
+          cpu: "1"
+          memory: 2Gi
+  protocolVersions:
+    - v1
+  supportedModelFormats:
+    - autoSelect: true
+      name: paddle
+      version: "2"
+---
+apiVersion: serving.kserve.io/v1alpha1
+kind: ClusterServingRuntime
+metadata:
+  name: kserve-pmmlserver
+spec:
+  annotations:
+    prometheus.kserve.io/path: /metrics
+    prometheus.kserve.io/port: "8080"
+  containers:
+    - args:
+        - --model_name={{.Name}}
+        - --model_dir=/mnt/models
+        - --http_port=8080
+      image: kserve/pmmlserver:v0.10.0
+      name: kserve-container
+      resources:
+        limits:
+          cpu: "1"
+          memory: 2Gi
+        requests:
+          cpu: "1"
+          memory: 2Gi
+  protocolVersions:
+    - v1
+  supportedModelFormats:
+    - autoSelect: true
+      name: pmml
+      version: "3"
+    - autoSelect: true
+      name: pmml
+      version: "4"
+---
+apiVersion: serving.kserve.io/v1alpha1
+kind: ClusterServingRuntime
+metadata:
+  name: kserve-sklearnserver
+spec:
+  annotations:
+    prometheus.kserve.io/path: /metrics
+    prometheus.kserve.io/port: "8080"
+  containers:
+    - args:
+        - --model_name={{.Name}}
+        - --model_dir=/mnt/models
+        - --http_port=8080
+      image: "<%= node['kube-hops']['docker_img_reg_url'] %>/<%= node['kube-hops']['sklearnserver']['image'] %>:<%= node['kube-hops']['docker_img_version'] %>"
+      name: kserve-container
+      resources:
+        limits:
+          cpu: "1"
+          memory: 2Gi
+        requests:
+          cpu: "1"
+          memory: 2Gi
+  protocolVersions:
+    - v1
+  supportedModelFormats:
+    - autoSelect: true
+      name: sklearn
+      version: "1"
+---
+apiVersion: serving.kserve.io/v1alpha1
+kind: ClusterServingRuntime
+metadata:
+  name: kserve-tensorflow-serving
+spec:
+  annotations:
+    prometheus.kserve.io/path: /metrics
+    prometheus.kserve.io/port: "8080"
+  containers:
+    - args:
+        - --model_name={{.Name}}
+        - --port=9000
+        - --rest_api_port=8080
+        - --model_base_path=/mnt/models
+        - --rest_api_timeout_in_ms=60000
+      command:
+        - /usr/bin/tensorflow_model_server
+      image: tensorflow/serving:2.6.2
```

Comment:
The TensorFlow version is set by Hopsworks during deployment creation, which replaces the version specified here. However, I would suggest we add the TensorFlow used in Hopsworks by default here anyway.
We have a Hopsworks variable for it, and I think also an attribute in one of the chef recipes.